blog/2021-05-10-eclipse-vert-x-4-1-Beta1-released.mdx

```diff
@@ -0,0 +1,137 @@
+
+## Vertx-Web
+
+In this release we improved the logging handler to allows custom logging formats.
+
+Oauth2 handler now follows the OIDC standard by validating the requested scopes are present in the oauth2 response, and a new APIKEY handler was added to support more security use cases and it is now possible to write custom authn handlers without requiring internal classes.
+
+The API `ctx.json()` now follows the IANA content type standard.
+
+## Vertx-Web OpenAPI
+
+A lot of community discussions and work has been done to improve the security handling of OpenAPI in vert.x. We now support all kinds of security described in OpenAPI 3.0, OpenId, Oauth2, API-Key, HTTP.
+
+The new API also allows fetching the security configuration from the document itself, rather than having it doubled configured in the document and in the code.
+
+## Vertx-Auth
+
+In vert.x auth module support for `EdDSA` was added to the JOSE/COSE code. This means that we can now use those algorithms in JWT/OAuth2/OIDC/FIDO2. The support relies only on the provided JDK which means that users will need to use JDK 15 or above to be able to use those algorithms.
+
+In the JOSE code, improvements were made to correclty follow the `use` value of a JWK and more tests were added to verify that we interop properly with tokens generated by other libraries.
+
+Finally in `FIDO2/Webauthn` we follow the recommendation of storing the attestation certificates, which allows offline validation of devices. This is a utility that allows to check if tokens have been compromised and disable their future use at the RP.
+
+## Web Session Storage with Infinispan
+
+Vert.x Web [session storage](https://vertx.io/docs/vertx-web/java/#_session_stores) is pluggable.
+In this version, a new session store with [Infinispan](https://infinispan.org/) in-memory data grid is available.
+
+It is implemented on top of the Infinispan client so it can be used with either standalone or clustered Vert.x applications.
+
+If you want to get started don't miss the [_Web Session Storage with Infinispan Client_ how-to](https://how-to.vertx.io/web-session-infinispan-howto/).
+
+## Finally
+
+This is the first beta release of Vert.x 4.1. You can of course expect more as we get feedback from the community and fix issues that we failed to catch before.
+
+The [deprecations and breaking changes](https://github.com/vert-x3/wiki/wiki/4.1.0-Deprecations-and-breaking-changes)
+ can be found on the wiki.
+
+For this release, there are no Docker images.
+
+The release artifacts have been deployed to [Maven Central](https://search.maven.org/search?q=g:io.vertx%20AND%20v:4.1.0.Beta1) and you can get the distribution on [Maven Central](https://repo1.maven.org/maven2/io/vertx/vertx-stack-manager/4.1.0.Beta1/).
+
+You can bootstrap a Vert.x 4.1.0.Beta1 project using https://start.vertx.io.
+
+The documentation has been deployed on this preview web-site: https://vertx.io/docs/
```

Comment:
preview web site?

maybe a wrong copy and paste here

Reply:
we still don't know yet how to handle documentation for beta